spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/system/version/EdcVersion.java

```diff
@@ -12,10 +12,9 @@
  *
  */
 
-package org.eclipse.dataspaceconnector.ids.spi.version;
+package org.eclipse.dataspaceconnector.spi.system.version;
 
-// TODO move to edc-spi
-public interface ConnectorVersionProvider {
+public interface EdcVersion {
```

Comment:
Why do we need all these interfaces and impls for a simple version string?

How about we let the `ConnectorServiceImpl` either use a hard-coded version string (`0.0.1-SNAPSHOT`) instead of the `ConnectorVersionProvider`, and we delete the `ConnectorVersionProvider` and the `EdcVersion`?
In the future we could improve this:
- create a gradle task that updates this hard-coded value before compile/build
- create a gradle task that generates a separate version file during build, which is then parsed by the `ConnectorServiceImpl`. That file would have to be shipped together with EDC code.

Reply:
@paullatzelsperger Changed as proposed